proxmin/algorithms.py

```diff
@@ -356,50 +337,43 @@ def bpgm(X0s, proxs_f, steps_f_cb, update='cascade', update_order=None, accelera
             steps_f_j = steps_f_cb(j, X)
 
             # acceleration?
-            if omega > 0:
+            # check for resizing: if resize ocurred, temporily skip acceleration
```

Comment:
Should we also reset omega? This is a whole new matrix now, and I'm not sure how that will affect the acceleration.